src/overview/basics.md

```diff
@@ -0,0 +1,27 @@
+# Git Driven Infrastructure
+
+As a Platform as a Service or PaaS Platform.sh manages automatically everything your application needs in order to run. 
+
+You don't need to setup anything manually. The web server is already setup and configured, as are any database, search engine or cache you might need.  
+
+Platform.sh covers not only all of your hosting needs but also most of your DevOps needs. It is a simple, single tool that covers the application life-cycle from development to production and scaling.
+
+![](https://platform.sh/sites/default/files/scalable-images/on-the-fly.png)
```

Comment:
We can have another representation of the "multiple environments" thing... but I think it is useful to represent this graphically here. It is not a trivial concept "git versioned infrastructure" or "branches of infrastructure"